[ews-build] Convert trivial shell.ShellCommand steps

JonWBedard opened the pull request:
#### 6ec9f3ac0e1a6301a8d59f11c66eebb36a3b40a0
<pre>
[ews-build] Convert trivial shell.ShellCommand steps
<a href="https://bugs.webkit.org/show_bug.cgi?id=256740">https://bugs.webkit.org/show_bug.cgi?id=256740</a>
rdar://109285960

Reviewed by Aakash Jain.

There are a number of shell.ShellCommand steps that can be trivially
converted to newstyle steps.

* Tools/CISupport/ews-build/steps.py:
(CheckOutSpecificRevision): Convert to shell.ShellCommandNewStyle steps.
(CleanWorkingDirectory): Ditto.
(RunBuildbotCheckConfig): Ditto.
(InstallGtkDependencies): Ditto.
(InstallWpeDependencies): Ditto.
(InstallBuiltProduct): Ditto.
(ArchiveBuiltProduct): Ditto.
(ExtractBuiltProduct): Ditto.
(ArchiveTestResults): Ditto.
(ApplyWatchList): Ditto.
(PushPullRequestBranch): Ditto.

Canonical link: <a href="https://commits.webkit.org/264089@main">https://commits.webkit.org/264089@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ed4a19e40742b0f91ab98c21bc2880a928a73c70

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/6495 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/6705 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/6885 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/8075 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/6785 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/6491 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/7035 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/6656 "Built successfully") | [❌ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/9681 "1 missing results 104 failures") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/6609 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/6666 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/5883 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/8155 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/4130 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/5861 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/13710 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/6066 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/5940 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/8281 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/6430 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/5261 "Passed tests") | | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/6456 "Passed tests") | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/5834 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/9995 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/776 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/6206 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

aj062 reviewed (approved):
rs=me. Please make sure this is tested.

JonWBedard committed:
[ews-build] Convert trivial shell.ShellCommand steps
https://bugs.webkit.org/show_bug.cgi?id=256740
rdar://109285960

Reviewed by Aakash Jain.

There are a number of shell.ShellCommand steps that can be trivially
converted to newstyle steps.

* Tools/CISupport/ews-build/steps.py:
(CheckOutSpecificRevision): Convert to shell.ShellCommandNewStyle steps.
(CleanWorkingDirectory): Ditto.
(RunBuildbotCheckConfig): Ditto.
(InstallGtkDependencies): Ditto.
(InstallWpeDependencies): Ditto.
(InstallBuiltProduct): Ditto.
(ArchiveBuiltProduct): Ditto.
(ExtractBuiltProduct): Ditto.
(ArchiveTestResults): Ditto.
(ApplyWatchList): Ditto.
(PushPullRequestBranch): Ditto.

Canonical link: https://commits.webkit.org/264089@main
